src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -27,6 +27,8 @@
     "1285": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "1287": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "4002": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "7341": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```

Comment:
This contract is not available
![image](https://user-images.githubusercontent.com/2896048/157626641-23df1f68-1ee4-4414-8777-6fade5a00d4c.png)

Reply:
Fixed, it seems we'd deployed 1.3.0 (not libs0) before. :
`
deploying "SignMessageLib" (tx: 0xc1f33a6aecfa039552da89873857b826385b368a906fb02615214fed61da2d9a)...: deployed at 0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2 with 311441 gas
`
https://bx.shyft.network/tx/0xc1f33a6aecfa039552da89873857b826385b368a906fb02615214fed61da2d9a/internal-transactions